site/content/docs/5.3/components/placeholders.md

```diff
@@ -83,7 +83,7 @@ We apply additional styling to `.btn`s via `::before` to ensure the `height` is
   <span class="placeholder col-6"></span>
 </p>
 
-<a href="#" tabindex="-1" class="btn btn-primary disabled placeholder col-4" aria-hidden="true"></a>
```

Comment:
Maybe drop `aria-hidden` too?

Reply:
Ah yes, I thought for no reason that is was `aria-disabled="true"`